readme.md

```diff
@@ -48,7 +48,7 @@ Milliseconds to humanize.
 
 #### options
 
-##### secDecimalDigits
+##### sec-decimal-digits
```

Comment:
Sure. Only for `secDecimalDigits` ? (`verbose` and `compact` are fine in their present form, right?)